crates/rolldown/tests/fixtures/tree_shaking/pure_annotation/_config.json

```diff
@@ -1,5 +1,9 @@
 {
   "config": {
-    "treeshake": true
+    "treeshake": {
+      "Option": {
+        "module_side_effects": true
```

Comment:
Please enable the `camelCase` for consistency.

Reply:
Done